unit_test/test_validate_plugin.py

```diff
@@ -227,6 +228,12 @@ def test_example_input_validator_should_fail_when_not_all_exists(self):
         result = validate(directory_to_test, file_to_test, False, True, [ExampleInputValidator()])
         self.assertEqual(result, 1)
 
+    def test_cloud_ready_connection_credential_token_validator_should_fail(self):
+        directory_to_test = "plugin_examples/cloud_ready_connection_credential_token_validator"
+        file_to_test = "plugin.spec.yaml"
+        result = validate(directory_to_test, file_to_test, False, True, [ConnectionValidator()])
```

Comment:
```suggestion
        result = validate(directory_to_test, file_to_test, False, True, [ConnectionValidator()])
```
Update to match new name

Reply:
Updated!